### PR TITLE
Sparelhoff/60 force drawer down

### PR DESCRIPTION
**Pull request by smparelhoff**
* Deleted old Drawer component/module
* Added new module: https://github.com/osdnk/react-native-reanimated-bottom-sheet
* Refactored Explore Brooklyn to use new bottom-sheet component
* Updated some styling to match what we had before with new component
* Implemented see more/see less functionality in Landmark component

### CODE REVIEW

File: screens/Landmark.js [hunk 1]
```diff
@@ -37,6 +43,33 @@ class LandmarkScreen extends React.Component {
...
+    this.props.closeDrawer();
+  };
+
+  getDescription = (type) => {
+    const { landmarkDetails } = this.props
+    let description = landmarkDetails.description;
+    let descriptionMaxWordLength = 500;
+    let shortDescriptionMaxWordLength = 100;
+
+    if (type === 'teaser') {
```
Comment (rockycodes) [inline screens/Landmark.js]: Heads up that eventually the short description will be a different chunk of text than the description, so we will need to refactor to use that once we have all the fields/data in the db. But this works great for now so we can test visually.

File: screens/Landmark.js [hunk 2]
```diff
@@ -116,18 +149,15 @@ class LandmarkScreen extends React.Component {
...
+        </View>
+      </ScrollView>
     );
   }
 }
 
 const mapDispatchToProps = dispatch => ({
   fetchDirections: coordinate => dispatch(getDirections(coordinate)),
-  clearDirections: () => dispatch(clearDirections())
+  eraseDirections: () => dispatch(clearDirections())
```
Comment (rockycodes) [inline screens/Landmark.js]: just curious why you changed this method name?

File: screens/Landmark.js (hunk 1 above)
Comment (smparelhoff) [reply screens/Landmark.js]: Yup, all good. Should be an easy thing to add.

File: screens/Landmark.js (hunk 2 above)
Comment (smparelhoff) [reply screens/Landmark.js]: It felt a little confusing since the method being imported from the store was also clearDirections.

**Pull request merged**